spec/classes/redis_sentinel_spec.rb

```diff
@@ -112,6 +122,13 @@
         it { is_expected.to compile.with_all_deps }
         it { is_expected.to create_class('redis::sentinel') }
         it { is_expected.to contain_file(config_file_orig).with_content(expected_content) }
+
+        if facts[:os]['family'] == 'Debian'
+          it { is_expected.to contain_package('redis-sentinel').with_ensure('latest') }
+        else
+          it { is_expected.not_to contain_package('redis-sentinel') }
+          it { is_expected.to contain_package('redis').with_ensure('latest') }
+        end
```

Comment:
Ensuring something is not managed is a bit error prone.  I would suggest something like:

```suggestion

        if facts[:os]['family'] == 'Debian'
          sentinel_package_name = 'redis-sentinel'
        else
          sentinel_package_name = 'redis'
        end
        it { is_expected.to contain_package(sentinel_package_name).with_ensure('latest') }
```

Reply:
My intention was to leave the original test case but I agree with you and that looks truly better. I'll update the change.